src/composables/sdk13.ts

```diff
@@ -150,6 +155,30 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
     return sdk;
   }
 
+  /**
+   * drySdk is the sdk instance with no accounts attached.
+   * To use for multisig operations.
+   */
+  async function getDrySdk(): Promise<AeSdk> {
+    if (!drySdk) {
+      const nodeInstance = await createNodeInstance(activeNetwork.value.url);
+      drySdk = new AeSdk({
+        nodes: [{
+          name: activeNetwork.value.name,
+          instance: nodeInstance!,
+        }],
+      });
+      return drySdk;
+    }
+    const networkId = await drySdk.api.getNetworkId();
```

Comment:
I'm not sure if fetching the dry SDK network ID every time we use `getDrySdk` is necessary. Maybe it's better to store the dry SDK network ID on the first initialization and then comparing it with the `nodeNetworkId` that comes from the main SDK?

Reply:
https://github.com/aeternity/superhero-wallet/pull/2140#discussion_r1243609216

Reply:
OK, I'm also surprised same as Subhod was. Weird that things under `api` property sometimes does not do fetching :O